packages/components/src/slot-fill/fill.ts

```diff
@@ -7,31 +7,26 @@ import { useContext, useLayoutEffect, useRef } from '@wordpress/element';
  * Internal dependencies
  */
 import SlotFillContext from './context';
-import useSlot from './use-slot';
 import type { FillComponentProps } from './types';
 
 export default function Fill( { name, children }: FillComponentProps ) {
 	const registry = useContext( SlotFillContext );
-	const slot = useSlot( name );
```

Comment:
The `Fill` no longer needs the `slot` value because it no longer calls `slot.rerender()`. Instead, it updates the value in the `fills` map (`updateFill` call) and the relevant slot rerenders automatically because it listens for `fills` changes.

Reply:
The fact that there were two separate `useSlot` hooks was definitely confusing. Glad to see that the "non-bubblesVirtually" version is going away

Reply:
The removed `useSlot` hook was essentially a `useSyncExternalStore` call. Now when the slot registry uses `observableMap`, it can be fully replaced by `useObservableMap`, which is also a thin wrapper around `useSyncExternalStore`.